### DISCUSSION TITLE
Question on XPath Query Fields

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#5703] [Q&A] [answered]
I'm starting to work with XPATH and FreshRSS (latest Docker Version) for sites that do not provide RSS feeds that I would like to scrap for data.  I'm slightly confused on what the "XPATH for Finding News Items" is asking for.  I can inspect and locate the DIV that is used for the grid having all the article objects within and I'm hoping to treat each as an item.  With the below configuration I'm able to ingest one article, but inside that are the other pieces.  Should this be the main div starting with the first article and go from there?  Any help would be welcomed as I know once I get it, I will be able to do other sites.  

Image of Site Query:
![Screenshot 2023-10-12 at 12 23 18 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/e136b361-7575-453a-9472-47183afacb42)

Site Inspect with Elements:
![Screenshot 2023-10-12 at 12 19 28 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/5734f88f-2a0c-478c-ab4d-56fd052ddfa0)

What I see in FreshRSS upon an Update:
![Screenshot 2023-10-12 at 12 33 25 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/30d41ee7-6c87-4bd8-9cad-4df94d050c8d)

**Comment by quella** [comment]
Sorry, the URL I'm trying to work with in my XPATH scrape is here: https://makerworld.com/de/models/categories?categories=0&orderBy=newUploads

**Reply by Frenzie** [reply]
Could it be you need to delete all articles/force reload?

**Reply by Frenzie** [reply, marked as answer]
Ah but no, that's simply the selector for their parent. You need `portal-css-0` instead.

**Reply by quella** [reply]
So, "portal-css-0" would be the pointer to the first item in the index containing the articles in the series?  So I would put this in the main "Most Important" field and from there the other article fields would be relative to the main "portal-css-0" for each one; correct?  I think you have me on the right track if I understand it correctly.  What a powerful tool once one understand what it can do.  Thank you for setting me on the right track.  Off to test.

**Reply by Frenzie** [reply]
It's not the first; it's all of them. ;-)

But basically yes.

**Reply by quella** [reply]
Roger roger.  Thanks for the confirmation sir.

**Comment by quella** [comment]
That solved it and thank you for getting my feet wet on this.  Now I'm off to tackle a few others as they are not a scary after your input.  Thank you.
